src/web/mod.rs

```diff
@@ -480,6 +480,12 @@ pub fn start_web_server(addr: Option<&str>, context: &dyn Context) -> Result<(),
 
     let axum_addr: SocketAddr = addr.unwrap_or(DEFAULT_BIND).parse()?;
 
+    log::info!(
```

Comment:
this should be `tracing::info` since we migrated

Reply:
Done!